docs/docs/toolkits.md

```diff
@@ -0,0 +1,198 @@
+# Toolkits
+
+This page contains information about building and using toolkits in Goose. Toolkits are a way to extend Goose's capabilities by adding new tools and functionalities. You can create your own toolkits or use the existing ones provided by Goose.
+
+## Using Toolkits
+
```

Comment:
Add a section using the `sq toolkit list` command to list the available toolkits you have